validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/AttestationProductionDutyTest.java

```diff
@@ -406,6 +444,11 @@ public void shouldCreateAttestationForMultipleValidatorsInDifferentCommittees()
                 unsignedAttestation2.getBeaconBlockRoot()),
             Optional.empty());
     verifyNoMoreInteractions(validatorLogger);
+
+    verify(validatorDutyMetrics, times(2))
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.CREATE));
+    verify(validatorDutyMetrics, times(3))
+        .record(any(), any(AttestationProductionDuty.class), eq(Step.SIGN));
```

Comment:
This is correct. 2 validators are in one committee, and the other is in a different committee. So we need to create 2 attestations and each validator will sign.